tutorials/Binary_Contingency_Scores.ipynb

```diff
@@ -0,0 +1,2738 @@
+    "# The default is operator.gt, which is the same as \">\" but in functional form.\n",
+    "\n",
+    "event_operator = scores.categorical.ThresholdEventOperator(default_event_threshold=1.3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "3447d68f-c7cb-40af-bdb0-42c5f88f4c43",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<xarray.DataArray (height: 2, lat: 3, lon: 3)> Size: 18B\n",
+      "array([[[False, False,  True],\n",
+      "        [False,  True,  True],\n",
+      "        [False, False,  True]],\n",
+      "\n",
+      "       [[ True, False,  True],\n",
+      "        [ True,  True, False],\n",
+      "        [ True,  True,  True]]])\n",
+      "Coordinates:\n",
+      "  * height   (height) int64 16B 10 20\n",
+      "  * lat      (lat) int64 24B 0 1 2\n",
+      "  * lon      (lon) int64 24B 5 6 7\n"
+     ]
+    }
+   ],
+   "source": [
+    "# The event operator creates binary \"event tables\". \n",
+    "# If you don't wish to visualise the binary \"event tables\", you can ignore this section.\n",
+    "# However, if you wish to visualise or utilise the the \"event tables\" directly, you can do so as follows:\n",
+    "\n",
+    "forecast_binary, observed_binary = event_operator.make_event_tables(simple_forecast, simple_obs)\n",
+    "print(forecast_binary)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dec28e21-6d2c-486c-8387-b6d164dce570",
+   "metadata": {},
+   "source": [
+    "## Step Two: Making Binary Contingency Information\n",
+    "\n",
+    "Once the event tables have been made, the next step is to make a contingency manager object. It is possible (and quicker) to make a contingency manager from the event operator directly, but we will first show the process as separate steps, and then show how to do the two things at once. The contingency manager stores the binary forecast event data, the binary observed event data and the contingency table itself, plus provides functions for aggregating the data along various dimensions, and calculating metrics from that information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "1cab2d73-ca4b-4c90-b0e2-db940c0cd198",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Contingency Manager (xarray view of table):\n",
+      "<xarray.DataArray (contingency: 5)> Size: 40B\n",
+      "array([ 9,  6,  2,  1, 18])\n",
+      "Coordinates:\n",
+      "  * contingency  (contingency) <U11 220B 'tp_count' 'tn_count' ... 'total_count'\n"
+     ]
+    }
+   ],
+   "source": [
+    "contingency_manager = scores.categorical.BinaryContingencyManager(forecast_binary, observed_binary)\n",
+    "print(contingency_manager)  # Print a view of the contingency table"
```

Comment:
Instead of printing the table, just do 
```
contingency_manager.get_table()
```
like you do below

Reply:
done